app/components/sy-checkmark/template.hbs

```diff
@@ -1 +1 @@
-<FaIcon ...attributes @icon={{this.icon}} />
+<FaIcon class="{{if @highlight "highlight"}}" ...attributes @icon={{this.icon}} />
```

Comment:
100%

Reply:
And you could use the component like this:
```hbs
<SyCheckmark  @checked={{some.nasty.deeply.nested.prop}} @highlight={{true}} />
``` 
instead of 
```hbs
<SyCheckmark  @checked={{some.nasty.deeply.nested.prop}} @highlight={{some.nasty.deeply.nested.prop}} />
```